src/Security/MemberAuthenticator/ChangePasswordHandler.php

```diff
@@ -272,13 +275,12 @@ public function doChangePassword(array $data, $form)
         $member->AutoLoginExpired = DBDatetime::create()->now();
         $member->write();
 
-        if ($member->canLogIn()) {
+        if ($member->canLogin()) {
```

Comment:
This is just an irrelevant PHPStorm complaining fix, irrelevant and not hurting the actual API. It just makes it case-sensitive.